clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -129,20 +130,27 @@ common_args = ["-key", "home_domain", args.domain,
                "-default_behaviors", "all,-bye",
                "-t", "t1"]
 
+if args.aka:
+    initial_reg_xml = "aka_initial_register.xml"
+    re_reg_xml = "aka_re_register.xml"
+else:
+    initial_reg_xml = "digest_register.xml"
+    re_reg_xml = "digest_register.xml"
```

Comment:
Do we gain anything here by using two variables instead of just the one variable, register.xml?